Gopkg.lock

```diff
@@ -2,201 +2,314 @@
 
 
 [[projects]]
+  digest = "1:5f8ec75d953827c5f7b5faf8bffd5573dfd65dd181f07e769c6681036fb5db75"
```

Comment:
As this change has nothing todo with dependencies I would prefer if you would remove the gopkg.lock & package-lock.json changes from the PR.